include/libtorrent/session.hpp

```diff
@@ -149,6 +149,8 @@ namespace libtorrent
 		libtorrent::dht_settings dht_settings;
 
 		dht::dht_storage_constructor_type dht_storage_constructor;
+
+		bdecode_node state;
```

Comment:
I was thinking of ``m_dht_state``: https://github.com/arvidn/libtorrent/blob/master/include/libtorrent/aux_/session_impl.hpp#L924